pkg/replica/replica.go

```diff
@@ -1156,3 +1177,72 @@ func (r *Replica) GetReplicaStat() (int64, int64) {
 
 	return lastModifyTime, headFileSize
 }
+
+func (r *Replica) GetFileInterval(includeBackingFileLayer bool) (sparse.GetFileInterval, error) {
+	if err := preload(&r.volume, includeBackingFileLayer); err != nil {
+		return nil, err
+	}
```

Comment:
I am thinking if we can avoid using `preload` here. Because the extent retrieval for all snapshots files in `preload` is duplicate with the following function `getFileInterval`.

Maybe you can:
    1. Move the main part of this function into `diff_disk.go`
    2. Directly iterate all snapshot files to get the replica fiemap. This may be similar to `diffDisk.lookup`

Reply:
I think preload is needed for the function `getFileInterval` to work correctly. Let's have a sync call about this.

Reply:
Discussed with Shuo